packages/manager/cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts

```diff
@@ -386,3 +396,115 @@ describe('linode landing checks', () => {
     cy.findByText('Created:').should('not.exist');
   });
 });
+
+describe('linode landing checks for empty state', () => {
+  beforeEach(() => {
+    const mockAccountSettings = accountSettingsFactory.build({
+      managed: false,
+    });
+
+    cy.intercept('GET', apiMatcher('account/settings'), (req) => {
+      req.reply(mockAccountSettings);
+    }).as('getAccountSettings');
+    cy.intercept('GET', apiMatcher('profile')).as('getProfile');
+    cy.intercept('GET', apiMatcher('linode/instances/*'), (req) => {
+      req.reply(emptyLinodeData);
+    });
```

Comment:
Unless the test needs to do something unusual, we'll usually favor the mocking and intercept utils in `support/intercepts` to set up these mocks.

In this case, the equivalent would be:

```suggestion
    mockGetAccountSettings(mockAccountSettings).as('getAccountSettings');
    interceptGetProfile().as('getProfile');
    mockGetLinodes([]);
```

Although I'm wondering if we really need the account setting mocks or profile intercept! It might be worth trying something like this instead:

```typescript
mockGetLinodes([]).as('getLinodes');
cy.visitWithLogin('/linodes');
cy.wait('@getLinodes');
```

Reply:
Made changes in this commit https://github.com/linode/manager/pull/10882/commits/dbc04a5811cc5c60a8e16a7c8c058708ec04025f